media/css/firefox/all.less

```diff
@@ -4,6 +4,14 @@
 
 @import "../sandstone/lib.less";
 
+@font-face {
```

Comment:
Again, let's try and avoid using icon fonts. They are heavy to download, especially when only using a single icon.